src/note/commitment.rs

```diff
@@ -33,29 +38,52 @@ impl NoteCommitment {
 impl NoteCommitment {
     /// $NoteCommit^Orchard$.
     ///
-    /// Defined in [Zcash Protocol Spec § 5.4.8.4: Sinsemilla commitments][concretesinsemillacommit].
+    /// Defined in [ZIP-226: Transfer and Burn of Zcash Shielded Assets][notecommit].
```

Comment:
Do not remove the existing reference; it is the correct reference for `$NoteCommit^Orchard$`. Instead reference both locations, and also mention `$NoteCommit^OrchardZSA$` in the docstring.